[DM-2164] Home Page rev2 doc update (#955)
SoftwareAG/c8y-docs: content/users-guide/device-management-bundle/overview.md

|[Managing device data](#managing-device-data)|How to manage [firmware](#firmware-repo) and [software](#software-repo) for devices, [configuration snapshots](#configuration-repository), [device credentials](#credentials), [trusted certificates](#trusted-certificates) and [device profiles](#device-profiles).
|[SmartREST templates](#smartrest-templates)|How to work with [SmartREST  templates](#smartrest-templates), a collection of request and response templates used to convert CSV data and {{< product-c8y-iot >}} Rest API calls.
|[Working with simulators](#simulator)|How to model devices with the [simulator](#simulator) in order to have the same level of functionality as connected hardware devices.
### Home dashboard

> The Home screen of the Device Management application is a dashboard which shows data for the general tenant.

Reviewer: what is the general tenant?

Author: The actual tenant ;) I have removed this

![Home dashboard](/images/users-guide/DeviceManagement/device-management-home-screen.png)

The data shown on the Home dashboard is shared by all users of the tenant. By default, the Home dashboard includes recent alarms and quick links.

> The Home dashboard can be edited and designed individually according to your needs. You can add, remove or change widgets being displayed here.

Reviewer: ```suggestion
The Home dashboard can be edited and designed individually according to your needs. You can add, remove or change widgets which are displayed here.
```

Author: Changed

> For details on editing a dashboard, refer to [Working with dashboards](#dashboards).

Author: Commited suggestions

> To reset the Home dashboard to its original content, click at the right of the top menu bar and then click **Restore dashboard**.

Reviewer: ```suggestion
To reset the Home dashboard to its default, click **Restore dashboard** at the right of the top menu bar. 
```

Author: Changed

|[Managing device data](#managing-device-data)|How to manage [firmware](#firmware-repo) and [software](#software-repo) for devices, [configuration snapshots](#configuration-repository), [device credentials](#credentials), [trusted certificates](#trusted-certificates) and [device profiles](#device-profiles).
|[SmartREST templates](#smartrest-templates)|How to work with [SmartREST  templates](#smartrest-templates), a collection of request and response templates used to convert CSV data and {{< product-c8y-iot >}} Rest API calls.
|[Working with simulators](#simulator)|How to model devices with the [simulator](#simulator) in order to have the same level of functionality as connected hardware devices.

> ### Home dashboard

Reviewer: ```suggestion

### Home dashboard
```
